actions/ctl_test.go

```diff
@@ -16,169 +16,338 @@ import (
+			checkTX: func(t *testing.T, tx *corazawaf.Transaction, logEntry string) {
+				if wantToContain, have := "[ERROR] Unknown toggle", logEntry; !strings.Contains(have, wantToContain) {
+					t.Errorf("Failed to log entry, want to contain %q, have %q", wantToContain, have)
+				}
+			},
 		},
-		{
-			input: "debugLogLevel=A",
-			check: func(t *testing.T, logEntry string, tx *corazawaf.Transaction) {
-				if wantToContain, have := "Invalid log level", logEntry; !strings.Contains(have, wantToContain) {
-					t.Errorf("unexpected log entry: want to contain %q, have %q", wantToContain, logEntry)
+		"responseBodyAccess too late": {
+			prepareTX: func(tx *corazawaf.Transaction) {
+				tx.ProcessRequestHeaders()
+				_, _ = tx.ProcessRequestBody()
+				tx.ProcessResponseHeaders(200, "HTTP/1.1")
+				_, _ = tx.ProcessResponseBody()
+			},
+			input: "responseBodyAccess=On",
+			checkTX: func(t *testing.T, tx *corazawaf.Transaction, logEntry string) {
+				if wantToContain, have := "[WARN] Cannot change response body access after response headers phase", logEntry; !strings.Contains(have, wantToContain) {
+					t.Errorf("Failed to log entry, want to contain %q, have %q", wantToContain, have)
+				}
+			},
+		},
+		"responseBodyAccess successfully": {
+			prepareTX: func(tx *corazawaf.Transaction) {
+				tx.ProcessRequestHeaders()
+			},
+			input: "responseBodyAccess=On",
+			checkTX: func(t *testing.T, tx *corazawaf.Transaction, logEntry string) {
+				if want, have := true, tx.ResponseBodyAccess; want != have {
+					t.Errorf("Failed to set responseBodyAccess, want %t, have %t", want, have)
+				}
+			},
+		},
+		"responseBodyLimit successfuly": {
+			input: "responseBodyLimit=12345",
+			prepareTX: func(tx *corazawaf.Transaction) {
+				tx.ProcessRequestHeaders()
+				_, _ = tx.ProcessRequestBody()
+				tx.ProcessRequestHeaders()
+			},
+			checkTX: func(t *testing.T, tx *corazawaf.Transaction, logEntry string) {
+				if tx.ResponseBodyLimit != 12345 {
+					t.Errorf("Failed to set response body limit, want %d, have %d", 12345, tx.ResponseBodyLimit)
+				}
+			},
+		},
+		"responseBodyLimit too late": {
+			input: "responseBodyLimit=12345",
+			prepareTX: func(tx *corazawaf.Transaction) {
+				tx.ProcessRequestHeaders()
+				_, _ = tx.ProcessRequestBody()
+				tx.ProcessResponseHeaders(200, "HTTP/1.1")
+				_, _ = tx.ProcessResponseBody()
+			},
+			checkTX: func(t *testing.T, tx *corazawaf.Transaction, logEntry string) {
+				if wantToContain, have := "[WARN] Cannot change response body access after response headers phase", logEntry; !strings.Contains(have, wantToContain) {
+					t.Errorf("Failed to log entry, want to contain %q, have %q", wantToContain, have)
+				}
+			},
+		},
+		"responseBodyLimit incorrect": {
+			input: "responseBodyLimit=a",
+			checkTX: func(t *testing.T, tx *corazawaf.Transaction, logEntry string) {
+				if wantToContain, have := "[ERROR] Incorrect integer value", logEntry; !strings.Contains(have, wantToContain) {
```

Comment:
Previous nit would require also: 
```suggestion
				if wantToContain, have := "[ERROR] Invalid limit", logEntry; !strings.Contains(have, wantToContain) {
```